src/sass/partials/components/_forms.scss

```diff
@@ -24,6 +24,34 @@
     }
   }
 
+  &--add {
+    @include button(setting-get('primary accent color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary accent color'), setting-get('primary color'), setting-get('primary color'), setting-get('primary color'));
+
+    background-image: url('/images/add.svg');
```

Comment:
We aren't going to be able to guarantee that this background image is always going to be there. This should be an inline SVG in the HTML